core/lib/spree/preferences/preferable.rb

```diff
@@ -101,6 +113,30 @@ def default_preferences
         ]
       end
 
+      # Preference types rendered as form fields in Solidus backend
+      #
+      # Not all preferences are representable as a form field.
+      #
+      # Arrays and Hashes for instance are supported preference field types,
+      # but do not represent well as a form field.
+      #
+      # As these kind of preferences are mostly developer facing
+      # and not admin facing we should not render them.
+      #
+      # Overwrite +allowed_admin_form_preference_field_types+ in your class that
+      # includes +Spree::Preferable+ if you want to provide more fields.
+      # If you do so, you also need to provide a preference field partial
+      # that lives in:
+      #
+      # +app/views/spree/admin/shared/preference_fields/+
+      #
+      # @return [Array]
+      def admin_form_preference_field_types
```

Comment:
Previously, I misread code in a future commit because this method includes `types`, prompting me to thing the result of this method is an array of `type` strings. Let's consider renaming it to `admin_editable_preferences`, so it communicates more clearly it returns an array of preferences?